l10n_it_fatturapa/models/account.py

```diff
@@ -29,7 +29,19 @@
 }
 
 
+class fatturapa_worksaddress(orm.Model):
+    #_position = ['1.2.3']
+    _name = "fatturapa.format"
```

Comment:
errore del copia incolla nel ordinamento model...  grazie elimino quello in riga 137 cosi rimangono ordinati per punti